orchestra/project_api/serializers.py

```diff
@@ -67,7 +66,18 @@ def get_status(self, obj):
         return dict(Task.STATUS_CHOICES).get(obj.status, None)
 
     def get_latest_data(self, obj):
```

Comment:
minor nitpick:

`-    """
-    Return latest input data for a specified task.
-
-    Args:
-        task (orchestra.models.Task):
-            The task object for which to retrieve data.
-
-    Returns:
-        latest_data (str):
-            A serialized JSON blob containing the latest input data.
-    """```